### PR TITLE
Fix e2e olm registry env vars for kibana and oauth-proxy

### PR DESCRIPTION
**Pull request by periklis**

### CODE REVIEW

File: olm_deploy/scripts/catalog-deploy.sh [hunk 1]
```diff
@@ -5,7 +5,7 @@ export IMAGE_ELASTICSEARCH_OPERATOR_REGISTRY=${IMAGE_ELASTICSEARCH_OPERATOR_REGI
 export IMAGE_ELASTICSEARCH_OPERATOR=${IMAGE_ELASTICSEARCH_OPERATOR:-registry.svc.ci.openshift.org/ocp/${OPENSHIFT_VERSION}:elasticsearch-operator}
 export IMAGE_ELASTICSEARCH6=${IMAGE_ELASTICSEARCH6:-registry.svc.ci.openshift.org/ocp/${OPENSHIFT_VERSION}:logging-elasticsearch6}
 export IMAGE_LOGGING_KIBANA6=${IMAGE_LOGGING_KIBANA6:-registry.svc.ci.openshift.org/ocp/${OPENSHIFT_VERSION}:logging-kibana6}
-export IMAGE_OAUTH_PROXY=${IMAGE_OAUTH_PROXY:-registry.svc.ci.openshift.org/ocp/${OPENSHIFT_VERSION}:elasticsearch-proxy}
+export IMAGE_OAUTH_PROXY=${IMAGE_OAUTH_PROXY:- registry.svc.ci.openshift.org/ocp/${OPENSHIFT_VERSION}:oauth-proxy}
```
Comment (jcantrill) [inline olm_deploy/scripts/catalog-deploy.sh:8]: We need both the oauth-proxy to front kibana and the elasticsearch-proxy to front ES

Comment (periklis) [reply olm_deploy/scripts/catalog-deploy.sh]: Indeed we have to add them both.

**Pull request merged**